doc_src/functional_hazard_analysis_longline/fha_longline.tex

```diff
@@ -355,34 +358,847 @@ \subsection{Phase 1: Approach and Positioning}
+	     
+%1.8
+\fhaTable[fhaSystemKeyValue=
+			{Leash},
+		 fhaFunctionKeyValue=
+		 	{load transfer},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{red}{high}},
+	     fhaNumberKeyValue=
+	     	{1.8},
+	     fhaCausesKeyValue=
+	     	{No causes for spontaneous leash rupture known}, 
+	     fhaDescriptionKeyValue=
+	     	{Spontaneous tearing of the leash due to contact with a sharp object under load.
+	      	This is generally completely eliminated by the construction of the normal leash made of climbing rope and enveloping tubular band with the same load capacity.
+	       	However, it is included in the analysis because a simple leash can be used for easier cutting}, 
+	     fhaMitigationKeyValue=
+	     {\tabitem Avoid sharp objects on harnesses/clothing of \textcolor{teal}{rescuer} and \textcolor{purple}{patient} \\
+	     \tabitem Additional self-belay loop that can be hung in the leash ring if required \\
+	     Leash shorter than the patient’s arm length so that the leash ring can be reached while sitting in a harness},
+	     fhaCritPostKeyValue=
+	     {\textcolor{red}{high}, but very unlikely. Even with a simple leash when using suitable clothing and harnesses and with caution.}]
+
+\clearpage
+
+A failure of the patient's and rescuer's climbing harness will not be discussed here, as this is considered unlikely when
+using current and undamaged harnesses.
+
+
+\subsection{Phase 2: Hanging the patient}
+\label{sec:fha:phase2}
+
+%2.1
+\fhaTable[fhaSystemKeyValue=
+			{helicopter},
+		 fhaFunctionKeyValue=
+		 	{load transfer},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{red}{high}},
+	     fhaNumberKeyValue=
+	     	{2.1},
+	     fhaCausesKeyValue=
+	     	{(Temporary) performance deficit of the helicopter}, 
+	     fhaDescriptionKeyValue=
+	     	{Before attaching the sling, this is described in 1.1. After attaching the sling (helicopter tied up), the rescuer's load can also be transferred tothe highline via the sling, \textcolor{purple}{patient} belt and leash if the helicopter accidentally loses altitude.
+  			If the helicopter has to leave its hovering position during this phase, the sling can be unloaded and unhooked.
+			Furthermore, the rope can be dropped and the \textcolor{teal}{rescuer} falls into the highline together with the \textcolor{purple}{patient}.
+    		However, the falling rope can pose a danger to both personnel and the highline (impact and friction on the line).}, 
+	     fhaMitigationKeyValue=
+	     	{\tabitem Abrasion-resistant coating of line and backups\\
+	     	\tabitem \textcolor{purple}{patient} and \textcolor{teal}{rescuer} wear an appropriate helmet\\
+	     	\tabitem Line construction with multiple backups made of core-sheath ropes\\
+	     	\tabitem Sufficient fall height under the highline for stretching by weight of two persons\\
+	     	\tabitem Terrestrial rescue equipment and an easily accessible highline stand\\
```

Comment:
not sure what a "stand" is in this context, should it be "anchor" instead ?
(the "highline stand" is used 3 times in doc so probably batch-replace)

Reply:
done in https://github.com/JBludau/Highline-Rescue-Project/pull/10/commits/b1ba87821e3153c4566d8fd80bf4880dbf32b427